### PR TITLE
Update the vip app command to follow the VIP-CLI style guide

### PR DESCRIPTION
**Pull request by yolih**
## Description

Updated the voice, tense, and format of command descriptions, examples, and usage models for the `vip app` command group to follow the VIP-CLI style guide.

## Pull request checklist

- [ n/a ] Update [SETUP.md](https://github.com/Automattic/vip-cli/blob/trunk/docs/SETUP.md#list-of-environmental-variables) with any new environmental variables.
- [ ] Update [the documentation](https://github.com/Automattic/vip-cli/blob/trunk/docs).
- [ ] [Manually test](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#manual-testing) the relevant changes.
- [ x ] Follow the [pull request checklist](https://github.com/Automattic/vip-cli/blob/trunk/docs/RELEASING.md#new-pull-requests)
- [ x ] Add/update [automated tests](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#automated-testing) as needed.

## New release checklist

- [ ] [Automated tests](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#automated-testing) pass.
- [ ] The [Preparing for release checklist](https://github.com/Automattic/vip-cli/blob/trunk/docs/RELEASING.md#preparing-for-release) is completed.

## Steps to Test

Observe the results of the `--help` display for impacted commands:

- `vip app`
- `vip app list`
- `vip app deploy`
- `vip app deploy validate`

### CODE REVIEW

**Review by rebeccahum — APPROVED** [review]

File: src/bin/vip-app-deploy-validate.ts [hunk 1]
```diff
@@ -45,21 +46,20 @@ export async function appDeployValidateCmd(
...
-	console.log( chalk.green( '✓ Compressed file has been successfully validated with no errors!' ) );
+	console.log( chalk.green( '✓ Compressed file has been successfully validated with no errors.' ) );
 }
 
 // Command examples for the `vip app deploy validate` help prompt
 const examples = [
 	{
-		usage: 'vip app @mysite.develop deploy validate <file.zip|file.tar.gz>',
-		description: 'Validate the compressed file to see if it can be deployed to your site',
+		usage: 'vip app deploy validate file.zip.tar',
```
Comment (sjinks) [inline src/bin/vip-app-deploy-validate.ts]: It's either `file.zip` or `file.tar.gz`.

File: src/bin/vip-app-deploy-validate.ts [hunk 2]
```diff
@@ -45,21 +46,20 @@ export async function appDeployValidateCmd(
...
+		description: 'Validate the directory structure of the local archived file named "file.tar".',
```
Comment (sjinks) [inline src/bin/vip-app-deploy-validate.ts]: File name does not match that in the "Usage".

File: src/bin/vip-app-deploy.ts [hunk 3]
```diff
@@ -252,27 +253,36 @@ Processing the file for deployment to your environment...
...
+		description:
+			'Deploy a local archived file named "file.zip" that contains application code to a VIP Platform environment that has Custom Deployment enabled.',
 	},
 	{
-		usage: 'vip app @mysite.develop deploy file.zip --message "This is a deploy message"',
-		description: 'Deploy the given compressed file to your site',
+		usage:
+			'WPVIP_DEPLOY_TOKEN=1234 vip @example-app.develop app deploy file.tar --message="A description for this deploy"',
+		description:
+			'Add a descriptive message for the Custom Deployment of the archived file named "file.tar".',
```
Comment (sjinks) [inline src/bin/vip-app-deploy.ts]: @rebeccahum does `vip app deploy` support plain `.tar` archives? Or do they have to be compressed (`.tar.gz`)?

Comment (sjinks) [reply src/bin/vip-app-deploy.ts]: I suppose it doesn't? https://github.com/Automattic/vip-cli/blob/966c5e858b3ae357a667012b690a7657990add03/src/lib/validations/custom-deploy.ts#L18

```suggestion
		usage:
			'WPVIP_DEPLOY_TOKEN=1234 vip @example-app.develop app deploy file.tgz --message="A description for this deploy"',
		description:
			'Add a descriptive message for the Custom Deployment of the archived file named "file.tgz".',
```

**Review by sjinks — CHANGES_REQUESTED** [review]

File: src/bin/vip-app-deploy.ts (hunk 3 above)
Comment (rebeccahum) [reply src/bin/vip-app-deploy.ts]: We do not support plain `.tar` archives. They have to be compressed.

File: src/bin/vip-app-deploy-validate.ts (hunk 2 above)
Comment (sjinks) [inline src/bin/vip-app-deploy-validate.ts]:
```suggestion
		usage: 'vip app deploy validate file.tar.gz',
		description: 'Validate the directory structure of the local archived file named "file.tar.gz".',
```

**Review by sjinks — APPROVED** [review]

File: src/bin/vip-app-deploy.ts (hunk 3 above)
Comment (yolih) [reply src/bin/vip-app-deploy.ts]: Thank you for catching this @sjinks & @rebeccahum ! ❤️

**Pull request merged**